### PR TITLE
Add a scatter property to some entities

### PR DESCRIPTION
**Pull request by cdonati**
This makes it easier to get a sample of keys for a given kind. It's used by the MapReduce library when finding split points:
https://github.com/GoogleCloudPlatform/appengine-mapreduce/wiki//ScatterPropertyImplementation

### CODE REVIEW

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/4091/
https://ocd.appscale.com:8080/job/Autoscale-test-manual-triggers/851/

File: AppDB/appscale/datastore/cassandra_env/cassandra_interface.py [hunk 1]
```diff
@@ -77,6 +78,12 @@
 # The size in bytes that a batch must be to use the batches table.
 LARGE_BATCH_THRESHOLD = 5 << 10
 
+# Entities have a .78% chance of getting the scatter property. Their keys
```
Comment (whoarethebritons) [inline AppDB/appscale/datastore/cassandra_env/cassandra_interface.py]: You didn't finish your

File: AppDB/appscale/datastore/cassandra_env/cassandra_interface.py [hunk 2]
```diff
@@ -77,6 +78,12 @@
...
+SCATTER_CHANCE = .0078
```
Comment (whoarethebritons) [inline AppDB/appscale/datastore/cassandra_env/cassandra_interface.py]: isn't 0.78% 0.000078?

Comment (cdonati) [reply AppDB/appscale/datastore/cassandra_env/cassandra_interface.py]: Representing a percentage as a decimal just requires dividing by 100.

Comment (whoarethebritons) [reply AppDB/appscale/datastore/cassandra_env/cassandra_interface.py]: had a feeling coffee didn't kick in for that comment.

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/4832/

**Comment by cdonati** [comment]
Demo:
With https://github.com/AppScale/hawkeye/pull/123: https://ocd.appscale.com:8080/job/Hawkeye/13915/

**Pull request merged**